LocalSettings.php

```diff
@@ -5311,6 +5311,7 @@
 		'crankipediawiki' => "//$wmgUploadHostname/crankipediawiki/4/4c/Crankilogo.png",
 		'cvmlabwiki' => "//$wmgUploadHostname/cvmlabwiki/3/3b/Wiki_logo.png",
 		'czechpolicywiki' => "//$wmgUploadHostname/czechpolicywiki/2/20/Plain_logo.png",
+		'dakhilcommunity' => "//$wmgUploadHostname/dakhilcommunity/4/49/Myfile.png",
```

Comment:
Note that the first part and ('dakhilcommunity') and the /dakhilcommunity/ in the link need to have "wiki" at the end.